static/templates/add_alert_word.hbs

```diff
@@ -0,0 +1,4 @@
+<form id="add-alert-word-form" class="form-horizontal">
+    <label for="add-alert-word-name">{{t "New alert word" }}</label>
+    <input type="text" name="alert-word-name" id="add-alert-word-name" class="required" maxlength=100 placeholder="{{t 'Alert word' }}" value="" />
+</form>
```

Comment:
Should we add this file at `templates/settings/`?

Reply:
Done